linera-service/src/cli_wrappers.rs

```diff
@@ -108,16 +107,16 @@ impl ClientWrapper {
 
     pub async fn project_new(&self, project_name: &str, linera_root: &Path) -> Result<TempDir> {
         let tmp = TempDir::new()?;
-        let mut command = self.run().await?;
+        let mut command = self.command().await?;
         command
             .current_dir(tmp.path())
-            .kill_on_drop(true)
             .arg("project")
             .arg("new")
             .arg(project_name)
             .arg("--linera-root")
-            .arg(linera_root);
-        assert!(command.spawn()?.wait().await?.success());
+            .arg(linera_root)
+            .spawn_and_wait_for_stdout()
+            .await?;
```

Comment:
(Ah, I looked for `panic!`, `unwrap` and `expect` but missed the `assert!`. :facepalm:)